cmd/uid_provider/api/controllers/base.go

```diff
@@ -0,0 +1,34 @@
+package controller
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"gitlab.inspr.dev/inspr/core/cmd/uid_provider/client"
+)
+
+// Server is a struct that contains the variables necessary
+// to handle the necessary routes of the rest API
+type Server struct {
+	Mux *http.ServeMux
+	Rdb client.RedisManager
+}
```

Comment:
Is this exposure necessary? I feel like these fields can be private

Reply:
You're right